Website/htdocs/mpmanager/app/Services/AgentTransactionService.php

```diff
@@ -51,6 +51,7 @@ static function ($q) use ($agentId) {
 
     public function getById(int $agentId, ?int $customerId = null): AgentTransaction
     {
+        /* @phpstan-ignore-next-line */
```

Comment:
Could you please change the ignore here to be more specific?

```suggestion
        // @phpstan-ignore larastan.noUnnecessaryCollectionCall
```

Applies to similar changes in this PR as well